### PR TITLE
[Dynamo] Fix ts and aot_ts backends with LazyGraphModule

### LINKED ISSUE #188872: [dynamo] ts and aot_ts backends broken by LazyGraphModule
### 🐛 Describe the bug

The `ts` and `aot_ts` backends fail on any function: `torch.jit.script` cannot script the `_LazyGraphModule` dynamo passes to backends since `use_lazy_graph_module` became the default (#117911). That PR wrapped the tests in `_force_skip_lazy_graph_module()`, so CI stays green while every user call fails.

```python
import torch

torch.compile(lambda x: torch.relu(x) + 1, backend="ts")(torch.randn(4))
```

Rebuilding a plain `GraphModule` before scripting fixes both backends, verified locally.

### Error logs

```
BackendCompilerFailed: backend='ts' raised:
RuntimeError:
method cannot be used as a value:
  File ".../torch/fx/_lazy_graph_module.py", line 120
        return cls.forward is cls._lazy_forward
               ~~~~~~~~~~~ <--- HERE
```

### Versions

```
PyTorch version: 2.14.0a0+gitc5bf344
OS: macOS 26.5.2 (arm64)
Python version: 3.11.13 (64-bit runtime)
CPU: Apple M1 Max
[pip3] torch==2.14.0a0+gitc5bf344
```

### PR DESCRIPTION
**Pull request by guan404ming**
## Related Issue

close #188872

## Why

Since `use_lazy_graph_module` became the default, dynamo hands backends a _LazyGraphModule that `torch.jit.script` cannot script, so ts and aot_ts failed on any function while tests masked it.

## How

- Rebuild a plain GraphModule before torch.jit.script in torchscript and ts_compile
- Drop _force_skip_lazy_graph_module() from tests so they cover the default config

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @kadeng @chauhang @amjames @jataylo @azahed98

### CODE REVIEW

**Comment by jansel** [comment]
@claude review these changes

**Comment by guan404ming** [comment]
Hi @jansel, I just addressed in latest push: factored the duplicated block into `_unwrap_lazy_graph_module` in `torch/fx/_lazy_graph_module.py` and switched both call sites to explicit imports (1, 2). No action on 3 since `strip_overloads` already mutated that graph before this PR. `test_torchscript_cpu` and `test_aot_ts_cpu` pass locally.

**Comment by ezyang** [comment]
@claude check the review comments are all fixed

**Review by jansel — APPROVED** [review]

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
## mergedog handoff

All CI is green (or skipped). Ready for human review and `@pytorchbot merge`.


Current PR head: `79e3e524dd1eafc6e79c2452329766c4ba9f27a2`.
No mergedog-authored commits were pushed during this run.

No LLM was invoked during this run.

**Comment by guan404ming** [comment]
Thanks!
